### PR TITLE
Login

### PR DESCRIPTION
**Pull request by blossomese**
Merging initial changes

### CODE REVIEW

**Review by blossomese — COMMENTED** [review]
I have reviewed these files, and they are good to go.

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,6 @@
...
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
```
Comment (ayuba-dn) [inline src/App.css:17]: I will suggest you use css variables for color codes. To enable dynamic theming

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,6 @@
...
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
```
Comment (ayuba-dn) [inline src/App.css:28]: use css variable here too

**Review by ayuba-dn — CHANGES_REQUESTED** [review]
Generally a nice work, kindly address the few comments

**Pull request closed without merging**